modbusApp/src/modbus.h

```diff
@@ -19,6 +19,7 @@
 #define MODBUS_WRITE_SINGLE_REGISTER         0x06
 #define MODBUS_WRITE_MULTIPLE_COILS          0x0F
 #define MODBUS_WRITE_MULTIPLE_REGISTERS      0x10
+#define MODBUS_REPORT_SLAVE_ID                 0x11
```

Comment:
@MarkRivers sorry for this misalignment, should I do another commit for aligning this?

Reply:
I will fix it after merging.